Providers/Translations.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
```

Comment:
@wolverinek14 Also separated the translation to a much more separated usage. Translations provider here, Dropdown in Components folder (used on Header by imported from there), and a Translation folder at app root level.